src/Pmi/Drc/RdrMetrics.php

```diff
@@ -52,4 +101,51 @@ public function metricsFields()
         $responseObject = json_decode($response->getBody()->getContents(), true);
         return $responseObject;
     }
+
+    /* Private Methods */
+
+    /**
+     * Get Date Range Bins
+     *
+     * Break up large date ranges segmented by maximum Metrics API 2 range
+     *
+     * @param string $start_date
+     * @param string $end_date
+     *
+     * @return array
+     */
+    private function getDateRangeBins($start_date, $end_date)
+    {
+        $date_range_bins = [];
+
+        $start = strtotime($start_date);
+        $end = strtotime($end_date);
+        $num_days_in_range = $end - $start;
+
+        // Metrics API 2 processes no more than 600 days of data per request
+        // Store in a smaller amount to avoid having > 1 MiB in Memcache
+        $max_days_for_metrics_api_2 = 200 * (24*60*60);
+
+        $num_bins = ceil($num_days_in_range / $max_days_for_metrics_api_2);
+
+        if ($num_bins == 1) {
+            array_push($date_range_bins, [$start_date, $end_date]);
+            return $date_range_bins;
+        }
+
+        $this_date = $start;
+
+        for ($i = 0; $i < $num_bins; $i++) {
+            $this_end_date = $this_date + $max_days_for_metrics_api_2;
```

Comment:
This ends up with overlapping dates at the end of one bin and the beginning of the next bin.  Which results in the merged data set containing duplicate data and counts being doubled at the bin boundaries.

![screen shot 2019-02-01 at 3 51 49 pm](https://user-images.githubusercontent.com/860499/52151618-5bffe880-2639-11e9-9be1-9e527304b26c.png)

Also, I'm worried about iterating by a number of seconds.  In general, if you're iterating by multiples (24*60*60) you're going to run into problems with DST.  This might not have any actual negative effects, but can you make sure?  The better solution would be to either use a DateTime object as an iterator and add intervals of days.  Or use strtotime (e.g. `strtotime('+200 days', $someTimestamp)`).